src/main/java/Duke.java

```diff
@@ -9,59 +10,72 @@
 
     public static void main(String[] args) {
         Scanner scan = new Scanner(System.in);
-        greet();
         boolean exit = false;
-        while (true) {
-            if (exit) {
-                break;
-            }
+        greet();
 
-            System.out.println("");
+        while (true) {
             String command = scan.nextLine();
-            String[] commandParts = command.split(" /");
-            String[] taskInfo = commandParts[0].split(" ", 2);
+            HashMap<String, String> commandArgs = parseCommand(command);
+            System.out.println(DIVIDER);
 
-            switch (taskInfo[0]) {
+            switch (commandArgs.get("command")) {
             case "list":
                 list();
                 break;
             case "done":
-                setTaskDone(Integer.parseInt(taskInfo[1]));
+                setTaskDone(Integer.parseInt(commandArgs.get("param")));
                 break;
             case "todo":
-                addTodo(taskInfo[1]);
+                addTodo(commandArgs.get("param"));
                 break;
             case "deadline":
-                String[] deadlineInfo = commandParts[1].split(" ", 2);
-                addDeadline(taskInfo[1], deadlineInfo[1]);
+                addDeadline(commandArgs.get("param"), commandArgs.get("by"));
                 break;
             case "event":
-                String[] eventInfo = commandParts[1].split(" ", 2);
-                addEvent(taskInfo[1], eventInfo[1]);
+                addEvent(commandArgs.get("param"), commandArgs.get("at"));
                 break;
             case "bye":
```

Comment:
What if use enters "Bye" would it work?

Reply:
Good catch, ill probably add in a lowercase modifier